client/state/sites/seo/reducer.js

```diff
@@ -0,0 +1,20 @@
+import { combineReducers } from 'redux';
```

Comment:
Small thing, but please don't forget to add the `external/internal` dependencies comment block. :)